theano/compile/builders.py

```diff
@@ -133,6 +135,59 @@ def perform(self, node, inputs, outputs):
             # we wont need this copy anymore
             output[0] = variable.copy()
 
+    def connection_pattern(self, node):
```

Comment:
Unless the existing implementation has issues, it should probably be simply reused; I don't think there's much point in changing code without a purpose.